pkg/apis/kops/instancegroup.go

```diff
@@ -103,6 +103,8 @@ type InstanceGroupSpec struct {
 	RootVolumeType *string `json:"rootVolumeType,omitempty"`
 	// If volume type is io1, then we need to specify the number of Iops.
 	RootVolumeIops *int32 `json:"rootVolumeIops,omitempty"`
+	// Keep the root volume upon instance termination
+	RootVolumeTermination *bool `json:"rootVolumeTermination,omitempty"`
```

Comment:
Naming suggestion: `retainVolumeOnTermination` or `deleteVolumeWhenInstanceTerminates` or something more self explanatory?

The `*bool` is good - it means that the flag will default to nil, and you don't have to make the existing behaviour  map to false - you can tell if someone explicitly set it or not

Reply:
@justinsb so with that in mind, the suggestion is to change the default volume termination to `false` if the field `retainVolumeOnTermination: true` exists?